### DISCUSSION TITLE
How to use this repo to analyze some web extensions?

### DISCUSSION THREAD

**Discussion by zydjohnHotmail** [webpack-contrib/webpack-bundle-analyzer#600] [Q&A] [answered]
Hello:
There are some web extensions I can get from Google store, they have the library built by webpack, as I can see the JS file contains something like this:
 (function webpackUniversalModuleDefinition(root, factory) ….
But there is no such file like ‘webpack.config.js’ or another useful information about the web extension.  But there are something else, like manifest.json and Popup.html, some CSS files, some folders, like “background”, “contentScript”, “inpage”.
There is one example you can download it from this URL: 

_LINK REDACTED BY @valscion, potentially malicious_

In this case, how can I use this repo to analyze the bundled JS file inside “background/lib” folder.
What I want to know is: for the bundled JS file, how many different modules are there to build the bundled JS, what are those modules names? If they are publicly available NPM packages, just show their npm packages.  In short, I want to use this repo to restore the “source code” which build the bundle with webpack.
Please advise,

**Comment by valscion** [comment, marked as answer]
Hi and thanks for the question!

This project requires you to have access to the [webpack Stats object](https://webpack.js.org/configuration/stats/).

The chrome extensions likely don't provide said object via a JSON file, so this project can't be used to analyze those bundles.
